app/src/org/odk/collect/android/widgets/GeoPointWidget.java

```diff
@@ -60,100 +59,86 @@ public GeoPointWidget(Context context, FormEntryPrompt prompt) {
 
         mWaitingForData = false;
         mUseMaps = false;
-        mAppearance = prompt.getAppearanceHint();
+        String appearance = prompt.getAppearanceHint();
+        // use maps or not
+        if (appearance != null && appearance.equalsIgnoreCase("maps")) {
+            try {
+                // do google maps exist on the device
+                Class.forName("com.google.android.maps.MapActivity");
```

Comment:
Wonder if we should set some application level "hasMaps" feature at startup since this is replicated in entityselectactivity